### PR TITLE
Matrix3GLf is now a class

### PR DESCRIPTION
**Pull request by StefanMermans**

### CODE REVIEW

File: ProftaakPeriode4/ProftaakPeriode4/GameObject.h [hunk 1]
```diff
@@ -1,10 +1,11 @@
 ﻿#pragma once
+
 #include "Matrix3GLf.h"
 #include <vector>
 
-
 class GameObject {
 	public:
+		// TODO: Bart... Document
```
Comment (bartmachielsen) [inline ProftaakPeriode4/ProftaakPeriode4/GameObject.h:8]: not blameble

**Review by bartmachielsen — APPROVED** [review]
OK

File: ProftaakPeriode4/ProftaakPeriode4/GameObject.h (hunk 1 above)
Comment (StefanMermans) [reply ProftaakPeriode4/ProftaakPeriode4/GameObject.h:8]: Your code, so you should document it.

**Comment by StefanMermans** [comment]
[(3) 3 lanes [3]](https://trello.com/c/78mY4cqY/7-3-3-lanes-3)

**Comment by StefanMermans** [comment]
[(3) 3 lanes [3]](https://trello.com/c/78mY4cqY/7-3-3-lanes-3)

**Pull request merged**